src/input/Input.js

```diff
@@ -133,6 +133,7 @@ Input.propTypes = {
   rightIconContainerStyle: ViewPropTypes.style,
 
   inputStyle: Text.propTypes.style,
+  inputComponent: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```

Comment:
Why is PropTypes.string used here?

Reply:
> `func` covers types from "classic" react components, ES6 classes, and stateless function components. `string` covers the native element case (eg "div").

From https://stackoverflow.com/a/33087573

Not sure if it does apply to react-native though, (maybe react-native-web?).

Reply:
Ah okay, wasn't sure.